### PR TITLE
 kie-kogito-docs-641: Remove Job Service All in one image from documentation

### LINKED ISSUE #641: Remove Job Service All in one image from documentation

### PR DESCRIPTION
**Pull request by wmedvede**
Closes #641 

**Description:**

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/apache/incubator-kie-kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `Issue-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] Issue-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Comment by wmedvede** [comment]
This is not draft anymore, would you mind guys review?
@ricardozanini @domhanak @nmirasch @kaldesai

**Comment by wmedvede** [comment]
Hi @kaldesai , this Job Service section is related to the conversation we had in Monday's meeting.
Where we chatted about that  Job Service should now go into the product documentation similar to what you did or are doing for the data-index.

**Review by ricardozanini — APPROVED** [review]

**Review by nmirasch — APPROVED** [review]

**Comment by domhanak** [comment]
@gmunozfe please check, this is JS related

File: serverlessworkflow/modules/ROOT/pages/job-services/core-concepts.adoc [hunk 1]
```diff
@@ -94,7 +79,7 @@ To configure the image by using environment variables you must pass one environm
 .Job Service image configuration for docker execution example
 [source, bash,subs="attributes+"]
 ----
-docker run -it -e JOBS_SERVICE_PERSISTENCE=postgresql -e VARIABLE_NAME=value {jobs_service_image_allinone}:latest
+docker run -it -e QUARKUS_DATASOURCE_USERNAME=postgresql -e VARIABLE_NAME=value {jobs_service_image_postgresql}:latest
```
Comment (gmunozfe) [inline serverlessworkflow/modules/ROOT/pages/job-services/core-concepts.adoc:82]: This value "postgresql" differs from the one in the example below. It's probably better to align them
```suggestion
docker run -it -e QUARKUS_DATASOURCE_USERNAME=postgres -e VARIABLE_NAME=value {jobs_service_image_postgresql}:latest
```

File: serverlessworkflow/modules/ROOT/pages/job-services/core-concepts.adoc [hunk 2]
```diff
@@ -49,42 +49,27 @@ Additionally, it can manage all the required configurations for every workflow t
...
+====
+The <<job-service-global-configurations, common configurations>> and the <<job-service-eventing-api, eventing API configurations>> are the same for both images.
+====
 
 We recommend that you follow this procedure:
 
-1. Identify the <<job-service-persistence,persistence mechanism>> to use and see the required configuration parameters.
+1. Identify the image to use depending on the persistence mechanism, and see the required configuration parameters specific for that image.
 2. Identify if the <<job-service-eventing-api, Eventing API>> is required for your needs and see the required configuration parameters.
 3. Identify if the project containing your workflows is configured with the appropriate xref:use-cases/advanced-developer-use-cases/job-service/quarkus-extensions.adoc#job-service-quarkus-extensions[Job Service Quarkus Extension].
```
Comment (gmunozfe) [inline serverlessworkflow/modules/ROOT/pages/job-services/core-concepts.adoc:70]: Wrong link
```suggestion
3. Identify if the project containing your workflows is configured with the appropriate xref:use-cases/advanced-developer-use-cases/quarkus-extensions.adoc#job-service-quarkus-extensions[Job Service Quarkus Extension].
```

**Review by gmunozfe — COMMENTED** [review]
Just align the value with the example and fix a wrong link not part of this PR

**Comment by wmedvede** [comment]
> Just align the value with the example

added

**Pull request merged**